keyboards/wuque/epiphany70/info.json

```
@@ -0,0 +1,690 @@
+{
+    "keyboard_name": "epiphany70",
+    "manufacturer": "wuque studio",
+    "maintainer": "spbgzh",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+      "bootmagic": true,
+      "command": false,
+      "console": false,
+      "extrakey": true,
+      "mousekey": true,
+      "nkro": true
```

rgblight_enable migrated from `rules.mk`
```suggestion
      "nkro": true,
      "rgblight": true
```